baybe/surrogates/ngboost.py

```diff
@@ -60,12 +61,23 @@ def _make_parameter_scaler_factory(
         # See base class.
 
         # Tree-like models do not require any input scaling
-        return
+        return None
+
+    @staticmethod
```

Comment:
I guess re-introducing is necessary because this is now `static`? Because the function looks identical to the one from the parent class

Reply:
what do you mean with "re-introduce"? Perhaps you overlooked: it overrides the return to `None`

Reply:
OK, then just overlooked that or checked at the wrong place.